python_modules/dagster/dagster/_core/definitions/asset_daemon_context.py

```diff
@@ -601,6 +602,7 @@ def build_run_requests_with_backfill_policies(
     asset_partitions: Iterable[AssetKeyPartitionKey],
     asset_graph: AssetGraph,
     run_tags: Optional[Mapping[str, str]],
+    dynamic_partitions_store: Optional[DynamicPartitionsStore] = None,
```

Comment:
can we switch `dynamic_partitions_store` to `instance_queryer: CachingInstanceQueryer`? I think we can make it non-optional (since it always exists), and name it `instance_queryer` instead